bokeh/core/property/containers.py

```diff
@@ -318,6 +319,12 @@ class PropertyValueColumnData(PropertyValueDict):
     def __setitem__(self, i, y):
         return self.update([(i, y)])
 
+    def __copy__(self):
+        return PropertyValueColumnData(copy.copy(dict(self)))
```

Comment:
I don't think the `copy.copy` is needed here, just calling the `dict` initializer creates a shallow copy

Reply:
I wasn’t sure if the result is exactly the same, but I’m hindsight you are likely right. With larger dicts it might make a difference in performance, I’ll remove the call to `copy.copy`.